storagenode/peer.go

```diff
@@ -390,6 +393,12 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 		)
 		pb.RegisterNodeGracefulExitServer(peer.Server.PrivateGRPC(), peer.GracefulExit.Endpoint)
 		pb.DRPCRegisterNodeGracefulExit(peer.Server.PrivateDRPC(), peer.GracefulExit.Endpoint)
+
+		peer.GracefulExit.Chore = gracefulexit.NewChore(
```

Comment:
it's looking like `peer.GracefulExit.Chore = gracefulexit.NewChore(` is in master already, so I'm a little confused how this one is different? I see that the one in master doesn't take satellite as an argument though. maybe this could be called `peer.GracefulExit.SatelliteChore` if it needs to be differentiated.

Reply:
the chore in this PR is in the storagenode/gracefulexit package.  what is in master is in the satellite/gracefulexit package. is that still weird?

Reply:
I think I just wasn't thinking big picture lol, I think this is fine as is!